cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -592,8 +592,7 @@ def test_ssa_compliance(local_setup_provider, ssa_compliance_profile, ssa_vm,
 @pytest.mark.rhv3
 @pytest.mark.tier(2)
 @pytest.mark.long_running
-@pytest.mark.meta(blockers=[BZ(1578792, forced_streams=['5.8', '5.9'],
-    unblock=lambda provider: vm_system_type != 'redhat')])
+@pytest.mark.meta(blockers=[GH('ManageIQ/integration_tests:8157')])
```

Comment:
@jan-zmeskal I can see this PR 8157 has been closed. can you please cross-check it..?

Reply:
Sorry for Typo :)

Reply:
@sbulage, good point. @jan-zmeskal, as far as I understand you have this issue only against RHV, right ?